### DISCUSSION TITLE
@ory/kratos-client 0.5.5-alpha.1 not published

### DISCUSSION THREAD

**Discussion by marcinkoziej** [ory/kratos#1008] [Q&A] [answered]
Hello,
I'd like to use Kratos 0.5.5  - it is also recommended in developer docs.
However, npmjs has no published version of typescript SDK: the last one is 0.5.4-alpha.1: https://www.npmjs.com/package/@ory/kratos-client

Dear Ory team,when is it expected to be able to use the TS sdk 0.5.5? I belive the flows API has change so I can't use sdk 0.5.4 with kratos 0.5.5.

**Comment by aeneasr** [comment, marked as answer]
Hey there, I'll be looking into this! Thank you for the report

**Reply by aeneasr** [reply]
Looks like the CI failed with a flaky job. I scheduled a re-run, let's see where it leads!

https://app.circleci.com/pipelines/github/ory/sdk/229/workflows/cd7a7cf9-83a2-4bb9-aa45-c1ee50796ffb/jobs/235

**Reply by aeneasr** [reply]
Unfortunately it looks like the problem is the spec: https://app.circleci.com/pipelines/github/ory/sdk/229/workflows/cd7a7cf9-83a2-4bb9-aa45-c1ee50796ffb/jobs/235

I won't be able to hot fix this at the moment. Is this an urgent issue to you?

The SDK 0.5.4 will work with 0.5.5 because there haven't been any breaking changes! We only do breaking changes from 0.5 -> 0.6 for example.
